code-scanning/properties/eslint.properties.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "ESLint",
+    "creator": "ESLint",
```

Comment:
Who is the POC for this?

Reply:
Me

Reply:
@yongyan-gh please remove this line as @josepalafox's team from GitHub would be the owners.

Reply:
removed the creator property